iceoryx2-ffi/c/src/api/attribute.rs

```diff
@@ -68,9 +68,6 @@ pub unsafe extern "C" fn iox2_attribute_key(
             buffer.cast(),
             copied_key_length,
         );
-        copied_key_length
-    } else {
-        0
```

Comment:
I'm wondering why this is removed. It might be useful to some of our users.

Reply:
No strong feelings about this, but others might have a use case. I'll leave it to you.